Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.5-alpine
 LABEL maintainer="Mostafa Hussein <mostafa.hussein91@gmail.com>"
 RUN apk add --no-cache gcc musl-dev libxml2-dev libxslt-dev nmap nmap-scripts openssl
 RUN pip install raccoon-scanner
-RUN adduser -D raccoon
+RUN groupadd -r -g 5055 raccoon && useradd -r -u 5055 -g raccoon raccoon
```

Comment:
`groupadd` call fails.. probably need to port to `addgroup` (and I assume the `adduser -> useradd` change will fail as well..